pytheus/metrics.py

```diff
@@ -104,6 +128,9 @@ def labels(self, _labels: Labels) -> 'Metric':
 
         return metric
 
+    def collect(self) -> Sample:
```

Comment:
I will take this chance to ask what you have against the `abc` module and abstract classes :slightly_smiling_face: